components/footer.js

```diff
@@ -180,7 +180,7 @@ const Footer = ({
             <Service href={`mailto:${email}`} icon="email-fill" name="Email" />
           </Grid>
           <Text my={2}>
-            <Link href="tel:1-855-625-4225">1-855-625-4225</Link>
+            <Link href="tel:1-855-625-HACK">1-855-625-HACK</Link>
```

Comment:
I'm switching this to still link to 4225, so at least the tooltip provides some clarity.


```suggestion
            <Link href="tel:1-855-625-4225">1-855-625-HACK</Link>
```